docs/Installation.md

```diff
@@ -51,6 +51,18 @@ here. *Pick another prefix at your peril!*
 
 ```sh
 mkdir homebrew && curl -L https://github.com/Homebrew/brew/tarball/master | tar xz --strip 1 -C homebrew
+eval "$(homebrew/bin/brew shellenv)"
+brew update --force --quiet
+chmod -R go-w "$(brew --prefix)/share/zsh"
+```
+
+or
+
+```sh
+git clone https://github.com/Homebrew/brew homebrew
+eval "$(homebrew/bin/brew shellenv)"
+brew update --force --quiet
+chmod -R go-w "$(brew --prefix)/share/zsh"
 ```
```

Comment:
```suggestion
```sh
git clone https://github.com/Homebrew/brew homebrew
#```

then 

```sh
eval "$(homebrew/bin/brew shellenv)"
brew update --force --quiet
chmod -R go-w "$(brew --prefix)/share/zsh"
#```
```

but remove the `#` before the trailing triple backticks